### PR TITLE
Add details to the README.

### PR DESCRIPTION
**Pull request by notlee**
Including:
- brief description taken from `package.json`
- contact information
- license information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+
 ## Usage
 
 ```shell
 npm init origami-component
 ```
+
+## Contact
+
+If you have any questions or comments about this component, or need help using it, please either [raise an issue](https://github.com/Financial-Times/o-subs-card/issues), visit [#origami-support](https://financialtimes.slack.com/messages/origami-support/) or email [Origami Support](mailto:origami-support@ft.com).		 If you have any questions or comments about this component, or need help using it, please either [raise an issue](https://github.com/Financial-Times/o-subs-card/issues), visit [#origami-support](https://financialtimes.slack.com/messages/origami-support/) or email [Origami Support](mailto:origami-support@ft.com).
```
Comment (JakeChampion) [inline README.md:13]:
```suggestion
If you have any questions or comments about this component, or need help using it, please either [raise an issue](https://github.com/Financial-Times/o-subs-card/issues), visit [#origami-support](https://financialtimes.slack.com/messages/origami-support/) or email [Origami Support](mailto:origami-support@ft.com).		 If you have any questions or comments about this component, or need help using it, please either [raise an issue](https://github.com/Financial-Times/.../issues), visit [#origami-support](https://financialtimes.slack.com/messages/origami-support/) or email [Origami Support](mailto:origami-support@ft.com).
```

Comment (notlee) [inline README.md:13]: Copy and paste got me 
```suggestion
If you have any questions or comments about this project, or need help using it, please either [raise an issue](https://github.com/Financial-Times/create-origami-component/issues), visit [#origami-support](https://financialtimes.slack.com/messages/origami-support/) or email [Origami Support](mailto:origami-support@ft.com).
```

Comment (JakeChampion) [reply README.md:13]: the link had o-subs-card, I replaced with ... as I have no idea what to replace with 🙈

File: README.md [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
+
+----
+
+## License
+
+Copyright (c) 2016 Financial Times Ltd. All rights reserved.
```
Comment (JakeChampion) [inline README.md:19]: that license year would be wrong, it should be the year they run the command?

Comment (notlee) [reply README.md:19]: copy paste got me again

File: README.md (hunk 1 above)
Comment (notlee) [reply README.md]: sorry I spotted and already resolved it, thanks for flagging!

File: README.md [hunk 3]
```diff
@@ -1,7 +1,21 @@
...
+
+## Contact
+
+If you have any questions or comments about this project, or need help using it, please either [raise an issue](https://github.com/Financial-Times/create-origami-component/issues), visit [#origami-support](https://financialtimes.slack.com/messages/origami-support/) or email [Origami Support](mailto:origami-support@ft.com).
+
+----
+
+## License
+
+Copyright (c) 2016 Financial Times Ltd. All rights reserved.
```
Comment (JakeChampion) [inline README.md:19]:
```suggestion
Copyright (c) 2020 Financial Times Ltd. All rights reserved.
```

Comment (notlee) [inline README.md:19]:
```suggestion
Copyright (c) 2020 Financial Times Ltd. All rights reserved.
```

**Pull request merged**